<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gdaxSetupAddWindow</class>
 <widget class="QWidget" name="gdaxSetupAddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add GDAX Account</string>
  </property>
  <widget class="QLineEdit" name="mApiKeyEntry">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>56</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Never Share bitProphet.dat</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>78</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Api Secret</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mApiSecretEntry">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>78</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="mSaveButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="mDefaultCheckbox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Make Default?</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="mAccountNameEntry">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>36</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="mCancelButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>38</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>58</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Api Key</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PassPhrase</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mApiPassPhrase">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
